app/models/samples/json/schedule_fixedinterval_increasing_fine_timestep.json

```diff
@@ -0,0 +1,158 @@
+{
+    "type": "ScheduleFixedIntervalAbridged",
+    "name": "Solstice Increasing",
+    "values": [
```

Comment:
@chriswmackey since there is no `is_leap_year` property, should I just add a validation to check whether the 24 <=`values` <= 8784

Reply:
@tanushree04 ,
I am only writing in the `is_leap_year` key when it is set to `True` since I think the vast majority of schedules just expect this to be `False`. So this means that we should make sure that the default value for `is_leap_year` in the schema is `False`.